NVDSuppressions.xml

```diff
@@ -10,4 +10,284 @@
 		</notes>
 		<cve>CVE-2021-43138</cve>
 	</suppress>
+	<suppress>
+		<notes>
+			<![CDATA[
+    file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+    See https://github.com/dotnet/announcements/issues/176 for full MS Details on CVE.
+    A remote code execution vulnerability exists when parsing certain types of graphics files.
+    This vulnerability only exists on systems running on MacOS or Linux.
+    This CVE applies to any .NET Core 3.1 but patch is provided at 3.1.405, ERP Facade is pinned to .Net core SDK > 6.0.101 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
+    ]]>
+		</notes>
+		<cve>CVE-2021-24112</cve>
+	</suppress>
+	<suppress>
+		<notes>
+			<![CDATA[
+      file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+        See https://github.com/dotnet/announcements/issues/112 for full MS Details on the CVE
+          A denial of service vulnerability exists when .NET Framework or .NET Core improperly handle web requests, aka '.Net Framework and .Net Core Denial of Service Vulnerability'. This CVE ID is unique from CVE-2019-0820, CVE-2019-0981.
+          Vunerability applies to System.Private.Uri version	4.3.0 and 4.3.1. System.Private.Uri is part of the .NetCore runtime, and fixed with version System.Private.Uri version 4.3.2.
+        FSS is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
+    ]]>
+		</notes>
+		<cve>CVE-2019-0980</cve>
+	</suppress>
+	<suppress>
+		<notes>
+			<![CDATA[
+      file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+        See https://github.com/dotnet/announcements/issues/113 for full MS Details on the CVE
+        Vunerability applies to System.Private.Uri version	4.3.0 and 4.3.1. System.Private.Uri is part of the .NetCore runtime, and fixed with version System.Private.Uri version 4.3.2.
+        The issue is fixed:         
+          2.1.11 or SDK 2.1.507 for Visual Studio 2017 or SDK 2.1.604 for Visual Studio 2019
+          2.2.5 or SDK 2.2.107 for Visual Studio 2017 or SDK 2.2.204 for Visual Studio 2019
+        ERP Facade is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
```

Comment:
? ERP Facade is pinned to .Net core SDK > 6.0.101

Reply:
Missed it, Thanks